src/index.html

```diff
@@ -13,6 +14,27 @@
       <div class='vertical-line'> </div>
       <img class='logo-img' src='assets/tcg_logo.png' alt='logo'>
     </div>
+    
+    <div class='features-decoration'> </div>
```

Comment:
this is something style related, the right way to do this is to use a pseudo-element in the `features-container` to  show the triangles. 

use a `::before` pseudo-element with position absolute using the triangle as background.

https://developer.mozilla.org/es/docs/Web/CSS/::before